charts/kube-state-metrics/templates/servicemonitor.yaml

```
@@ -11,6 +11,7 @@ metadata:
   {{- end }}
 spec:
   jobLabel: {{ default "app.kubernetes.io/name" .Values.prometheus.monitor.jobLabel }}
+  {{ include "servicemonitor.scrapeLimits" .Values.prometheus.monitor | indent 2 }}
```

Should it be `{{- '? Otherwise it will have 4 indents.
```suggestion
  {{- include "servicemonitor.scrapeLimits" .Values.prometheus.monitor | nindent 2 }}
```